index.html

```diff
@@ -144,5 +144,25 @@ <h3 class="subtitle">Skills</h3>
             </ul>
         </section>
     </section>
+
+    <section class="contactContainer">
+        <h2 class="title blue">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form class="contactForm" action="https://formspree.io/f/mzbwkvzy" method="POST">
+            <input class="singleLineInput" name="name" type="text" placeholder="Full Name" maxlength="30" required>
+            <input class="singleLineInput" name="email" type="email" placeholder="Email address" required>
+            <textarea class="commentArea" name="comments" placeholder="Enter text here" maxlength="500" required></textarea>
+            <button class="button">
+                Get in touch
+            </button>
+        </form>
+        <ul class="mediaContainer">
+            <li><a href="https://twitter.com/Victor_GonMa" target="_blank" ><img class="mediaIcon" src="images\twitter.png" alt="Twitter"></a></li>
+            <li><a href="https://www.linkedin.com/in/victor-manuel-gonzalez-massimi-a77265124/" target="_blank" ><img class="mediaIcon" src="images\Linkedin icon.png" alt="Twitter"></a></li>
+            <li><a href="https://github.com/Vgonma" target="_blank" ><img class="mediaIcon" src="images\medium.png" alt="Twitter"></a></li>
+            <li><a href="https://github.com/Vgonma" target="_blank" ><img class="mediaIcon" src="images\github.png" alt="Twitter"></a></li>
+            <li><a href="https://github.com/Vgonma" target="_blank" ><img class="mediaIcon" src="images\angellist.png" alt="Twitter"></a></li>
+        </ul>
+        <p>myemailhere@email.com</p>
+    </section>
```

Comment:
- [ ] Kindly replace the section tag, which wraps the contact form, with the `footer tag` or wraps all the section within the `footer tag`. This is the requirement for this milestone of the project. Please fix this :wrench: 
***Project requirement***
![Screenshot from 2022-11-02 00-55-44](https://user-images.githubusercontent.com/65129228/199350335-931a8329-9670-4505-884b-4bba7ad12678.png)

Example
```
<footer>
   <section>
    </section>
</footer
```
or

```
<footer>
   <div>
    </div>
</footer
```